src/clients/add-on_auto.rb

```diff
@@ -194,8 +194,11 @@ def main
                   )
                 )
               else
-                # just report error
-                Report.Error(_("Failed to add add-on product."))
+                # just report an error
+                error_string = format(_("Failed to add product \"%s\" via\n%s\n"),
```

Comment:
Please add a `TRANSLATORS:` comment, the meaning of `%s` placeholders is not obvious.